files/build_templates/enable_counters.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Enable SONiC counters
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/counterpoll queue enable
```

Comment:
Can we enable counters for all types, including queue, port and pfcwd?
with  /usr/bin/counterpoll queue enable /usr/bin/counterpoll port enable, /usr/bin/pfcwd counter_poll enable? 
The latter pfcwd counter requires following to be taken into build image

https://github.com/Azure/sonic-swss/commit/72a7f3a5cb99cec297c7eb0e9e3b2b583eccf8bd

https://github.com/Azure/sonic-utilities/pull/273

Reply:
Sure. I created a PR https://github.com/Azure/sonic-buildimage/pull/1814 for master branch. After we test it in the master I'll update this PR